litmus-portal/cluster-agents/subscriber/pkg/objects/objects.go

```diff
@@ -0,0 +1,297 @@
+package objects
```

Comment:
I think we don't need these functions. There is already a generic function in k8s pkg to do the CRUD operations.

Reply:
https://github.com/litmuschaos/litmus/blob/master/litmus-portal/cluster-agents/subscriber/pkg/k8s/operations.go

You can use/refactor operations.go acc to the usecase